core/object/script_language_extension.h

```diff
@@ -234,6 +234,16 @@ class ScriptLanguageExtension : public ScriptLanguage {
 		}
 	}
 
+	GDVIRTUAL0RC(Vector<String>, _get_doc_comment_delimiters)
+
+	virtual void get_doc_comment_delimiters(List<String> *p_words) const override {
+		Vector<String> ret;
+		GDVIRTUAL_REQUIRED_CALL(_get_doc_comment_delimiters, ret);
```

Comment:
This should probably use `GDVIRTUAL_CALL()` (rather than `GDVIRTUAL_REQUIRED_CALL()`), because otherwise this will generate an error when using a GDExtension that was created before this virtual method existed.